### ISSUE TITLE
The problem of training accuracy of 0

### ISSUE THREAD

**Issue by 15733171319** [SysCV/MaskFreeVIS#9]
Author, Hello, Thank you for your wonderful project on VIS. I want to know why I don 't put pre-training weights when I train on maskfree. As the number of training steps increases, the accuracy becomes lower and lower, and eventually becomes 0. Do you know why ? In addition, even if the pre-training weight is added, as long as the network is slightly improved, the network training accuracy is ultimately 0 ? What is the reason, thank you !

**Comment by lkeab** [comment]
Hi, training VIS models typically adopts a coco pre-trained model weights, either using coco masks + boxes or only using coco boxes. A good initial starting point will make the VIS model easy to converge and learn in the video training stage, especially in the weakly-supervised setting (where model parameters are very sensitive). To obtain the correct accuracy, please follow my training instructions/configs. Some other github users had already verified the replicated accuracy as in [here](https://github.com/SysCV/MaskFreeVIS/issues/4).

**Comment by 15733171319** [comment]
Author, hello, I want to reproduce your maskfree, but during the training process, if you do not put the pre-training weight, try many times, the segmentation accuracy is 0, this is why, I want to ask you, and I train weak supervised video instance segmentation network should use that pre-training weight, thank you!

**Comment by lkeab** [comment]
it should use pre-trained weights on image-based object boxes or masks
